### ISSUE TITLE
SS_output does not seem to read ageselex correctly

### ISSUE THREAD

**Issue by hgerritsen** [r4ss/r4ss#550]
I am using SS version 3.30.17.01 with R version 4.0.2 and r4ss_1.40.1
When I run SS_output(), the names of the ageselex dataframe are not read in correctly:
> names(replist$ageselex)
 [1] "COMBINED_ALK*selL*selA*wtlen*ret*discmort_in_makefishsel_yr:" "2019"                                                        
 [3] "With_MeanSel_From:"                                           "2019"                                                        
 [5] "-"                                                            "2019;"                                                       
 [7] "Year_styr-3_(1983)_stores_average_used_for_benchmark"         "NoName"                                                      
 [9] "NoName"                                                       "NoName"                                                      
[11] "NoName"                                                       "NoName"                                                      
[13] "NoName"                                                       "NoName"                                                      
[15] "NoName"                                                       "NoName"                                                      
[17] "NoName"                                                       "NoName"                                                      
[19] "NoName"                                                       "NoName"                                                      
[21] "NoName"                                                       "NoName"                                                      
[23] "NoName" 

The names are actually in the first row of the dataframe (i.e. it starts to read the data from the text file one row to soon):                                                     
> replist$ageselex[1,]
1441                                                       Factor Fleet                 Yr Seas Sex Morph
1441                                                Label      0      1      2      3      4      5      6      7      8      9
1441     10     11     12     13     14     15

When I compare my report file with one generated by the previous version of SS, there is indeed an extra row ("Aret_is_age_retention") which I presume is the cause of this problem:

 AGE_SELEX report:32
Asel_is_age_selectivity_alone
Asel2_is_sizesel*size_at_age(ALK)
**Aret_is_age_retention**
COMBINED_ALK*selL*selA*wtlen*ret*discmort_in_makefishsel_yr: 2019 With_MeanSel_From: 2019 - 2019; Year_styr-3_(1983)_stores_average_used_for_benchmark
Factor Fleet Yr Seas Sex Morph Label  0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15
Asel 1 1983 1 1 1 1983_1Asel  0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0

I have attached my input files for reference.
[input files.zip](https://github.com/r4ss/r4ss/files/7097335/input.files.zip)

**Comment by kellijohnson-NOAA** [comment]
Thank you for taking the time to report your issue here on the issue tracker. I was able to download your files, run the latest version of SS, and use the github version of r4ss, i.e.,
```
remotes::install_github("r4ss/r4ss")
```
to read in your files and I get
```
names(replist$ageselex)
[1] "Factor" "Fleet"  "Yr"     "Seas"   "Sex"    "Morph"  "Label"  "0"      "1"     
[10] "2"      "3"      "4"      "5"      "6"      "7"      "8"      "9"      "10"    
[19] "11"     "12"     "13"     "14"     "15"    
```

So, the fastest solution would be to use the latest version of r4ss available here on github, which is version 1.43.0. Let us know if you have problems installing it or if you have additional problems using it. If there are no further problems, then feel free to close the issue.
Best,
Kelli

**Comment by hgerritsen** [comment]
That worked
Thanks for your quick response.
